docs/reference.rst

```diff
@@ -252,13 +252,53 @@ Example configuration:
   It has no effect for ``plain`` bundles, as the signature verification already checks the
   whole bundle.
 
-``prevent-late-fallback=<true/false>`` (optional)
+``prevent-late-fallback=<true/false>`` (deprecated, optional)
+  .. note:: Values *<true/false>* for this option are deprecated. They will be removed in a future release
+    The syntax has changed. Please use the new values.
+    In order to get the same behaviour set ``prevent-late-fallback=mark-bad``.
+
+``prevent-late-fallback=<disabled/mark-bad/lock-counter>`` (optional)
   In some use-cases, fallback to an older version must be prevented after the
   update is completed successfully ('rauc status mark-good' executed from the
   new version).
-  If this option is enabled, RAUC will execute the equivalent of 'rauc status
-  mark-bad other' after marking the currently booted slot as good.
-  This means that the other slot(s) is/are no longer eligible for fallback.
+
+  Supported values are:
+
+  * ``disable`` Default if not set. Do not prevent fallback to older version.
+  * ``mark-bad`` If this option is enabled, RAUC will execute the equivalent of
+    'rauc status mark-bad other' after marking the currently booted slot as good.
+    This means that the other slot(s) is/are no longer eligible for fallback.
+    It behaves exactly like the *deprecated* ``prevent-late-fallback=false``.
+  * ``lock-counter`` This is a slot independent setting that preserves
```

Comment:
`prevent-late-fallback` is always slot-independent, no matter which value you give.